pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -301,4 +323,21 @@ public static void removeDsmField(String esIndex, String ddpParticipantId, Strin
         UpsertPainless upsert = new UpsertPainless(esIndex, QueryBuilders.termsQuery("_id", ddpParticipantId));
         upsert.export(script, Map.of(), field);
     }
+
+    public static void addParticipantDsmFromFile(String esIndex, String fileName, String ddpParticipantId) {
```

Comment:
Looks like a comment from my prior review got lost, but...
This should be combined with `addDsmEntityFromFile`, rather than creating a new method that does a similar thing differently.
If it is unclear how to do that, I can suggest a way.

Reply:
Unclear why this is now resolved.

Reply:
This method is removed and I am using `ElasticTestUtil.addDsmEntityFromFile(esIndex, "elastic/dsmKitRequestShipping.json", ddpParticipantId, "1990-10-10", null);` I think this addresses your concern but please let me know if I resolved it by mistake